cpp/src/worker.cpp

```diff
@@ -204,7 +204,11 @@ void Worker::initBlockingProgressMode()
                              }};
 
   err = epoll_ctl(_epollFileDescriptor, EPOLL_CTL_ADD, _workerFileDescriptor, &workerEvent);
-  if (err != 0) throw std::ios_base::failure(std::string("epoll_ctl() returned " + err));
```

Comment:
Strings are arrays so this was doing pointer arithmetic (AKA, array indexing).

Reply:
> Strings are arrays so this was doing pointer arithmetic (AKA, array indexing).

What about `std::string("epoll_ctl() returned " + std::to_string(err))`? Seems a bit less verbose and I _think_ still correct.

Reply:
Oh yeah, that would likely work...